website/docusaurus.config.js

```diff
@@ -151,6 +195,7 @@ const config = {
         href: "/",
       },
       items: [
+        { to: "blog", label: "Blog", position: "left" }, // or position: 'right'
```

Comment:
Can we add a "Doc" label before Blog so that users can navigate to the docs quickly?

Reply:
good idea